src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,21 +36,24 @@
  */
 public class RobotContainer {
   
-  // The robot's subsystems and commands are defined here...
+  //Example
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
```

Comment:
I know you mentioned that use 'm_' is a convention, but there's no need for it. You're only decreasing the readability of you're source code by adding this unnecessary symbol/identifier to everything. If it's actually serving a purpose, and helps you as the programmer read it, then ignore me, but for the sake of readability, I would personally not use it